### PR TITLE
Fixed Single/Multiple Vaccine Uploads;

### PR DESCRIPTION
**Pull request by filipbudd**
Fixed an issue where, if you upload only one vaccine pdf, it would save the file in the Google Cloud Storage as only the first letter of the vaccination type. (For example, rabies would be Last_Dog_r.pdf). This was caused by the name of the vaccination type being split into a list, with the first letter being located at the first position of the array, rather than the whole string itself. (This wasn't an issue with uploading multiple files, because the first position in the array was occupied with the full string of the first vaccine. (Intended behaviour.) This is what happens when I don't test my code as thouroughly as I'd like to, I guess.)

TL;DR - Now you can upload one, or multiple different vaccine DOGuments (documents, haha). A small but crucial change... Please accept my pull request we need to go live in two hours 🥺🙏

### CODE REVIEW

**Review by kvnbanunu — APPROVED** [review]
Nice code.
